packages/terra-time-input/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased
 
 ## 4.45.1 - (March 1, 2022)
+  * Added
```

Comment:
I think this should be moved to the Unreleased section.

Reply:
changed 1f38c2b4bd18f2546c492b916080c2f2b61b7917